query/src/main/java/org/infinispan/query/stats/impl/LocalIndexStatistics.java

```diff
@@ -36,7 +37,8 @@
                Class<?> javaClass = e.javaClass();
                SearchScope<?> scope = session.scope(javaClass, e.name());
                long count = session.search(scope).where(SearchPredicateFactory::matchAll).fetchTotalHitCount();
-               return new IndexInfo(count, 0);
+               long size = e.indexManager().unwrap(LuceneIndexManager.class).computeSizeInBytes();
+               return new IndexInfo(count, size);
```

Comment:
Thanks @yrodiere , this was not forgotten really, I was planning to send a follow up PR to decouple it from the upgrade itself, but you saved me some time  :+1: